pkg/service/one2onerestore/model.go

```diff
@@ -0,0 +1,72 @@
+// Copyright (C) 2025 ScyllaDB
+
+package one2onerestore
+
+import (
+	"github.com/pkg/errors"
+	"github.com/scylladb/scylla-manager/v3/pkg/scyllaclient"
+	. "github.com/scylladb/scylla-manager/v3/pkg/service/backup/backupspec"
+)
+
+// Target specifies what data should be restored and from which locations.
+type Target struct {
+	Location        []Location    `json:"location"`
+	Keyspace        []string      `json:"keyspace,omitempty"`
+	SourceClusterID string        `json:"source_cluster_id"`
+	SnapshotTag     string        `json:"snapshot_tag"`
+	NodesMapping    []nodeMapping `json:"nodes_mapping"`
+	BatchSize       int           `json:"batch_size,omitempty"`
```

Comment:
I don't think batch size is needed, it's ok to call RClone to copy all SSTables of the given keyspace/table of a given node at once.
RClone tracks the progress on its own, SM just calls for the progress using jobID.